stack/indexer/rpm/build_package.sh

```diff
@@ -158,6 +156,8 @@ main() {
         esac
     done
 
+    set -ex
```

Comment:
Remove this.

Reply:
This debugging tool is necessary for the Jenkins builds